styles.css

```diff
@@ -0,0 +1,511 @@
+  color: #172b4d;
+}
+
+.about-supporting-text {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  padding-left: 12px;
+  padding-right: 12px;
+  text-align: center;
+  color: #42526e;
+}
+
+.button-resume {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 24px;
+  padding-bottom: 59px;
+}
+
+.get-resume {
+  width: 159px;
+  height: 48px;
+  align-items: center;
+  right: 50%;
+  padding: 12px;
+  background: #36b37f;
+  border: none;
+  border-radius: 4px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+.language-detail .get-resume a {
+  color: #344563;
+  text-decoration: none;
+}
+
+.language-detail .get-resume:active {
+  color: #dfe1e6;
+  background: #52bee9;
+}
+
+.language-detail .get-resume:visited {
+  color: #dfe1e6;
+  background: #008552;
+}
+
+.language-detail .get-resume button:enabled {
+  background: #36b37f;
+}
+
+.language-detail .get-resume:hover {
+  background: #36b37f;
+}
+
+.back-image-2 {
+  background-image: url(image/illustration-\ about\ me.png);
+  background-size: contain;
+  background-repeat: no-repeat;
+}
+
+.back-image-3 {
+  background-image: url(image/illustration\ -\ about\ me-mobile.jpg);
+  background-size: contain;
+  background-repeat: no-repeat;
+  left: 305px;
+  display: flex;
+  flex-direction: row;
+  position: absolute;
+  width: 90.87px;
+  height: 100.94px;
+}
+
+.about-cards {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(352px, 1fr));
+  align-items: center;
+  padding: 1px;
+  gap: 24px;
+  width: 327px;
+  height: 1106px;
+}
+
+.language-detail {
+  width: 327px;
+  height: 352px;
+  background: #ebf0ee;
+  border: 1px solid #c1c7d0;
+  border-radius: 8px;
+}
+
+.language-detail .language-image {
+  width: 100px;
+  height: 100px;
+  margin-top: 40px;
+  margin-right: 50%;
+  margin-left: 123px;
+  margin-bottom: 0 auto;
+  left: calc(50% - 80px / 2 + 0.5px);
+  top: calc(50% - 80px / 2);
+}
+
+.language-detail div {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  gap: 24px;
+  width: 327px;
+  height: 96px;
+  left: calc(50% - 327px / 2);
+  bottom: 72px;
+}
+
+.language-detail h4 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  text-align: center;
+  color: #344563;
+}
+
+.language-detail ul {
+  display: flex;
+  flex-direction: row;
+  padding: 8px;
+  list-style: none;
+  align-items: center;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.language-detail li {
+  background: white;
+  color: #36b37e;
+  padding: 8px;
+  text-align: center;
+  border-radius: 8px;
+  font-size: 14px;
+  font-weight: 600;
+  line-height: 40px;
+  margin: 0 10px 10px 0;
+  letter-spacing: 0.03em;
+}
```

Comment:
[OPTIONAL]
- [ ] There is a lot of spacing to the right. Kindly fix that since that is not the case on the design template.

Your Design | Figma Design
--|--
![image](https://user-images.githubusercontent.com/29071984/198675899-a515fe20-b910-4e40-b4a3-fe4ede303824.png) | ![image](https://user-images.githubusercontent.com/29071984/198676418-d6697d01-4c9a-49be-a6af-0db1e9155784.png)